slack.tex

```diff
@@ -1,7 +1,77 @@
 \subsection{Slack}
 
-Description of the types of communications that happen on Slack
+The data management group uses instant messaging tools to answer questions, 
+share code snippets and errors, and coordinate activities in real-time.  
+Before using instant messaging, we would communicate primarily via e-mail or 
+over the phone. While these tools do still have their place, being able to have 
+conversations in a team-wide application has been very helpful. Discussions
+happen with more team members participating, instead of just who happened to
+be on a phone call or an e-mail thread. Questions and concerns have been able
+to be answered much more quickly.  Site channels help coordinate discussions
+useful to local teams.  We even have non-work related channels were we can
+discuss various topics, which has sparked conversations about previously
+unrealized common interests. It has been beneficial to team building and 
+helped foster friendships.
 
-Collaboration involvement
+We have used two different instant messaging tools in the last four years. The
+first tool we used was HipChat. Besides the essential features expected
+from an instant messaging client (such as discussion channels and one
+to one direct messaging), we were also able to receive Jenkins and JIRA issue
+notifications, which we found very helpful.
+
+We used HipChat for two years and eventually started looking at other options 
+because we found the user experience was, at times, inconsistent and 
+frustrating. One of the most significant issues was how it managed unread
+messages. If the user left the application, HipChat did not maintain where the
+user was in the message chain. When the tool restarted, it required the user
+to scroll back through messages to pick up where they left off. Notifications
+were erratic, and sometimes would arrive late or not at all. Some users
+particularly liked features such as video chat, while other users found this
+feature unstable and resorted to using other video chat services such as Google 
+Hangouts. 
+
+After a brief testing period, we decided to transition to Slack, which we
+continue to use today. The move from HipChat to Slack was straightforward,
+although we did lose the discussion history from HipChat.  At first, we 
+thought this would be an issue, but in practice, we rarely search history older
+than a couple of weeks.  We have had difficulties in balancing when to 
+archive these discussions for the community.  We've also had some problems
```

Comment:
By "community" I meant "when to formally migrate a discussion to community.lsst.org where a decision can be more easily disseminated."

Reply:
Before I commit this, is this more like what you'd like to see?  "We have had difficulties in balancing when to formally migrate discussions made in these channels so we can more easily disseminate decisions to community.lsst.org."

Reply:
Something like that. We can edit later. The point being that sometimes the five people in the channel come up with some consensus on some interesting topic but you can't leave it sitting there in the channel history. You either need to make a comment on Jira or write a community post summarizing the discussion.